### PR TITLE
Retyping of page0020.txt

### PR DESCRIPTION
**Pull request by sebras**

### CODE REVIEW

**Comment by larsbrinkhoff** [comment]
Oops, I thought you added new files rather than modifications to the old.  I see now there are some changes I might not agree with.

I have found that the line format is:
- A field of eight characters with a left-justified three-digit line number.
- A field of eight characters with two three-digit number for a symbol reference.
- Tab stops every eight character for mnemonics and comments.
- The top right page number is at 80 columns (10 tab stops).

Some of your changes don't seem to match this.  Do you disagree?

**Comment by larsbrinkhoff** [comment]
I haven't used any tabs, no.  Maybe we should do that.  But it can be changed later, if so.

File: transcribed/page0020.txt [hunk 1]
```diff
@@ -1,33 +1,33 @@
-001                                                                             PAGE 2
```
Comment (larsbrinkhoff) [inline transcribed/page0020.txt:1]: According to my cut-and-paste method, I have determined that the PAGE on line 1 is at column 80.  If you disagree, please provide some evidence.

File: transcribed/page0020.txt [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
-021     002 019 UTCON==2001\200_<-UTCCHN>\200_<-TTYCHN>\200_<-GETYOC>\200_<-DISCHN>\200_<-SDCHN>
-022     002 019 UTCOFF==1001\200_<-UTCCHN>\200_<-TTYCHN>\200_<-GETYOC>\200_<-DISCHN>\200_<-SDCHN>
-023     002 014 CUINT==5000+UTCCHN
-024             SCLKI==15.      ;SLOW CLOCK PER REG CLOCK
-025             MXOPT==8.       ;SLOW CLOCK  MAX UT OP CAN TAPE
-026             NINFP==8.       ;MAX. NO. OF INF. PROCED. / PROCED.
+010              LUPDL==30       ;LENGTH USER PDL
+011              CPDLL==20       ;LENGTH CLOCK CHANNEL PDL
+012              DCCHN==1        ;DC CHANNEL
+013      002 012 DCCHNA==200←<-DCCHN>
```
Comment (larsbrinkhoff) [inline transcribed/page0020.txt:13]: Please use _ instead of ←.

**Comment by larsbrinkhoff** [comment]
Good fixes, thanks!  With some exceptions as noted.  I'll merge and fix afterwards.

File: transcribed/page0020.txt (hunk 1 above)
Comment (sebras) [reply transcribed/page0020.txt:1]: nope, I believe you. I just didn't notice it or know about the file format at the time.

**Pull request merged**